airgun/widgets.py

```diff
@@ -701,6 +701,48 @@ def search(self, value):
             self.search_button.click()
 
 
+class PF4Search(Search):
+    """PF4 Searchbar for table filtering"""
+
+    ROOT = '//div[@role="combobox" or @aria-haspopup="listbox"]'
```

Comment:
Are we sure this is a PF4 component?

Looking at the PF4 reference components, I'm not seeing any use of either of these attributes in the search box div.

The PF4 Search component is in beta still, but uses the class `pf-c-search-input` for the root div. 

The text input field uses the class `pf-c-search-input__text-input`.

Reply:
This definitely needs to be revisit.  Maybe we can meet and discuss.